### DISCUSSION TITLE
Classic SF2 Joy

### DISCUSSION THREAD

**Discussion by CaptainGordon** [Dinir/mini-padder#18] [Custom Skins]
Classic Street Fighter 2 joystick layout.

- - -

**Preview**

![csf2](https://user-images.githubusercontent.com/9097044/93563590-30bf3e00-f9c3-11ea-87be-e1ed34bb87af.png) ![csf2_white](https://user-images.githubusercontent.com/9097044/93563593-3157d480-f9c3-11ea-9060-b696d5268bd0.png)
https://youtu.be/hCe-ZFvytkY

**Skin File**

- Archive: [Classic.SF2.Joy.zip](https://github.com/Dinir/mini-padder/files/7830560/Classic.SF2.Joy.zip), [Classic.SF2.Joy.White.zip](https://github.com/Dinir/mini-padder/files/7830559/Classic.SF2.Joy.White.zip)
- Single File: [Classic.SF2.Joy.txt](https://github.com/Dinir/mini-padder/files/7830536/Classic.SF2.Joy.txt), [Classic.SF2.Joy.White.txt](https://github.com/Dinir/mini-padder/files/7830537/Classic.SF2.Joy.White.txt)

**Comment by Dinir** [comment]
I included screenshots and single file versions for easier access, and brought the monochrome variant here.
Thank you very much for the contribution!

**Comment by CaptainGordon** [comment]
🙌 🎉
